src/main/java/org/breedinginsight/brapi/v2/model/request/query/ListQuery.java

```diff
@@ -17,14 +18,16 @@ public class ListQuery extends BrapiQuery {
     private String name;
     private String description;
     private String size;
+    private ExternalReferenceSource externalReferenceSource;
```

Comment:
I don't think this should be the enum that we have internally, but should be a `String`.  This would make the query params BrAPI compliant

Reply:
ok, made the change